client/src/main/java/com/battleships/gamewindow/board/Fleet.java

```diff
@@ -14,7 +18,45 @@ public void clear() {
         ships.clear();
     }
 
-    public String toString(){
+    public String toString() {
         return ships.toString();
     }
+
+    public int size() {
+        return ships.size();
+    }
+
+    public boolean isHit(Coordinate... coordinate) {
+        return Arrays.stream(coordinate)
+                .anyMatch(coord -> ships.stream()
+                        .flatMap(ship -> ship.getMasts().stream())
```

Comment:
Can you change 
.flatMap(ship -> ship.getMasts().stream())

to

.map(ship -> ship.getMasts())

? (Idk)

Reply:
flatmap is ok here